.github/workflows/create-tag.yaml

```diff
@@ -14,7 +14,7 @@ defaults:
 jobs:
   Create-Tag:
     runs-on: ubuntu-18.04
-    if: github.repository == 'rook/rook' && contains(secrets.RELEASE_ADMINS, github.actor)
```

Comment:
@travisn  you can delete `RELEASE_ADMINS` from the repo secrets

Reply:
also, we need to backport this to 1.7 and 1.6 right?

Reply:
Ok `RELEASE_ADMINS` is removed